src/components/movieInfo/styles.ts

```diff
@@ -0,0 +1,41 @@
+import {StyleSheet} from 'react-native';
+
+import {theme} from '@theme/themes';
```

Comment:
why `'@theme/themes'` and not `'@theme'` directly?

Reply:
I have a file themes.ts inside theme folder which has all the colors and other styles defined